vips/vip-401/bsctestnet.ts

```diff
@@ -0,0 +1,177 @@
+  [USDT_PRIME_CONVERTER]: USDT,
+  [USDC_PRIME_CONVERTER]: USDC,
+  [WBTC_PRIME_CONVERTER]: WBTC,
+  [WETH_PRIME_CONVERTER]: WETH,
+  [XVS_VAULT_CONVERTER]: XVS,
+};
+
+enum ConversionAccessibility {
+  NONE = 0,
+  ALL = 1,
+  ONLY_FOR_CONVERTERS = 2,
+  ONLY_FOR_USERS = 3,
+}
+
+export const vip401 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-401",
+    description: `LBTC`,
+    forDescription: "Process to configure and launch the new market",
+    againstDescription: "Defer configuration and launch of the new market",
+    abstainDescription: "No opinion on the matter",
+  };
+
+  return makeProposal(
+    [
+      // Configure Oracle
+      {
+        target: REDSTONE_ORACLE,
+        signature: "setDirectPrice(address,uint256)",
+        params: [marketSpec.vToken.underlying.address, parseUnits("1.1", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            marketSpec.vToken.underlying.address,
+            [LBTC_ONE_JUMP_REDSTONE_ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
+            [true, false, false],
+          ],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      // Add Market
+      {
+        target: marketSpec.vToken.underlying.address,
+        signature: "faucet(uint256)",
+        params: [marketSpec.initialSupply.amount],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: marketSpec.vToken.underlying.address,
+        signature: "approve(address,uint256)",
+        params: [POOL_REGISTRY, marketSpec.initialSupply.amount],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: [REDUCE_RESERVES_BLOCK_DELTA],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            marketSpec.vToken.address,
+            marketSpec.riskParameters.collateralFactor,
+            marketSpec.riskParameters.liquidationThreshold,
+            marketSpec.initialSupply.amount,
+            marketSpec.initialSupply.vTokenReceiver,
+            marketSpec.riskParameters.supplyCap,
+            marketSpec.riskParameters.borrowCap,
+          ],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: marketSpec.vToken.underlying.address,
+        signature: "approve(address,uint256)",
+        params: [POOL_REGISTRY, 0],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [marketSpec.riskParameters.protocolSeizeShare],
+        dstChainId: LzChainId.sepolia,
+      },
```

Comment:
This is unneeded, because the `protocolSeizeShare` is already set to `0.05e18` in the VToken contract